src/System.Private.CoreLib/shared/System/Text/UTF8Encoding.cs

```diff
@@ -154,12 +160,17 @@ public override unsafe int GetByteCount(char[] chars, int index, int count)
 
         public override unsafe int GetByteCount(string chars)
         {
-            // Validate input
-            if (chars==null)
-                throw new ArgumentNullException("s");
+            // Validate input parameters
+
+            if (chars is null)
```

Comment:
I'm seeing identical codegen for `== null` and `is null` on string.  Why the change?

Reply:
Cf. https://github.com/dotnet/coreclr/pull/21948#discussion_r273295021

Reply:
@gfoidl I'm seeing that both patterns just reduce to br.true or br.false from the compiler (with both string and array on single or multiple arg).  So while it may be true on types overloading operator== other than string, I'm challenging @tannergooding's assertion that it matters.

Once it doesn't matter, we're back to what I once saw on a poster: "change is terrible, unless it's awesome".  In this case, if there are two ways of identically doing the same thing, the one with fewer lines of diff is better.

Reply:
This is copied + pasted from the `ASCIIEncoding` class, which is acting as the source of truth for these implementations and will eventually roll out to `UnicodeEncoding`, `UTF32Encoding`, etc. It's more lines in this particular diff, but it's overall less diff if you compare __ASCIIEncoding.cs__ and __UTF8Encoding.cs__ against each other.